### PR TITLE
Site structure—from site map

### PR DESCRIPTION
**Pull request by misscs**

### CODE REVIEW

**Comment by misscs** [comment]
This is ready for review and merge.

In short, I went through the entire sitemap and wireframed out what we outlined. In the process, I added new tasks for components/features that we said we wanted but hadn't capture as issues. 

After seeing the sitemap in action, I'm not sure how I feel about `blog` being in `resources` section. Something for us to discuss. Maybe with more content it will make sense. 

The main goal was to give us a bigger picture of what this thing looks like, what's done, and what's missing/still a todo

File: assets/stylesheets/components/_team.scss [hunk 1]
```diff
@@ -1,42 +1,44 @@
-.team {
-
-	.member {
-		@include box;
-		float: left;
-		width: (100% / 3);
-		padding: 10px;
-		text-align: center;
-		border-left: solid 1px #ccc;
-		border-top: solid 1px #ccc;
```
Comment (adekunleoduye) [inline assets/stylesheets/components/_team.scss]: This layout portion of this needs to be refactor

Comment (misscs) [inline assets/stylesheets/components/_team.scss]: Refactored with flexbox?

Comment (adekunleoduye) [inline assets/stylesheets/components/_team.scss]: yes and we need to use vars for colors...but his can be merge this and worry about that late

Comment (misscs) [inline assets/stylesheets/components/_team.scss]: Flexbox is already noted as an issue #58 for you :P 

As for colors, waiting for the finals from the styleguide and I don't think this has the new color management system.

As a whole, I expect a ton of this has to be refactored. It is really just quick and dirty structural layout aimed to give a high level overview of what we envisioned.

**Pull request merged**